### PR TITLE
[Fluid Clean] move fluid.layers.deformable_conv to static.nn

### PR DESCRIPTION
**Pull request by zoooo0820**
### PR types

Others
### PR changes

APIs
### Describe

According to clean the code in paddle.fluid. The useless codes should be removed and code still be used should move to a new position. 

For function `paddle.fluid.layers.deformable_conv`, since it is still be used by `paddle.static.nn.deformable_conv2d`, it should be moved to the module `static.nn`.

### CODE REVIEW

**Review by jeff41404 — DISMISSED** [review]
LGTM

File: python/paddle/static/nn/common.py [hunk 1]
```diff
@@ -167,6 +174,251 @@ def fc(
...
+            deformable conv layer. If it is set to False, no bias will be added
+            to the output units. If it is set to None or one attribute of ParamAttr, conv2d
+            will create ParamAttr as bias_attr. If the Initializer of the bias_attr
+            is not set, the bias is initialized zero. Default: None.
+        modulated (bool): Make sure which version should be used between v1 and v2, where v2 is \
+            used while True. Default: True.
+        name(str, Optional): For details, please refer to :ref:`api_guide_Name`.
+                        Generally, no setting is required. Default: None.
+    Returns:
+        Variable: The tensor variable storing the deformable convolution \
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]: 我们现在还有variable的概念吗？

File: python/paddle/static/nn/common.py [hunk 2]
```diff
@@ -167,6 +174,251 @@ def fc(
...
+                  result. A Tensor with type float32, float64.
+    Raises:
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]: raises删掉

File: python/paddle/static/nn/common.py [hunk 3]
```diff
@@ -167,6 +174,251 @@ def fc(
...
+        ValueError: If the shapes of input, filter_size, stride, padding and
+                    groups mismatch.
+    Examples:
+        .. code-block:: python
+
+          #deformable conv v2:
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py:295]: 这里代码片段没有增加缩进，我担心会有编译问题，构建一个预览看看吧

File: python/paddle/static/nn/common.py [hunk 4]
```diff
@@ -167,6 +174,251 @@ def fc(
...
+    groups=None,
+    deformable_groups=None,
+    im2col_step=None,
+    param_attr=None,
+    bias_attr=None,
+    modulated=True,
+    name=None,
+):
+    r"""
+    :api_attr: Static Graph
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]: 这个删了吧，没啥意义

File: python/paddle/static/nn/common.py (hunk 1 above)
Comment (zoooo0820) [reply python/paddle/static/nn/common.py]: 已修改对应描述。
同时如下图预览所示，英文文档没有展示对应内容

image

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Review by jeff41404 — APPROVED** [review]

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**